NuGet.config

```diff
@@ -21,6 +21,7 @@
     <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
     <add key="vs-impl-archived" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl-archived/nuget/v3/index.json" />
     <add key="vs-buildservices" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
```

Comment:
This will be removed before merging. Need it temporarily while packages are published to internal feeds.

Reply:
Consider adding a `TODO` comment so our scanner blocks it.

Reply:
Good suggestion. Will do that for future changes like this.